src/clusterfuzz/_internal/bot/fuzzers/centipede/engine.py

```diff
@@ -88,6 +89,32 @@ class Engine(engine.Engine):
   def name(self):
     return 'centipede'
 
+  def _get_arguments(self, fuzzer_path):
+    """Gets the fuzzer arguments.
+    This function will fetch for an existing `.options` file and add some
```

Comment:
Change to this returns default arguments and arguments specified byt the options filed.